### PR TITLE
[14.0][MIG] stock_move_line_auto_fill

### PR DESCRIPTION
**Pull request by ivantodorovich**
Standard migration

### CODE REVIEW

**Review by dsolanki-initos — COMMENTED** [review]
Code looks good.
Please follow the Oca guideline for commits.You need to change the sequence of commits here.

Functional Test:
For the products without tracking it is working fine.
For the products with lot "Auto Fill" functionality seems not working.

Please have a look at the video link I have prepared
https://www.awesomescreenshot.com/video/3187440?key=c42c6ab8b14d50af55ab99ffae5f49c8

**Comment by ivantodorovich** [comment]
> Code looks good.
> Please follow the Oca guideline for commits.You need to change the sequence of commits here.

Thanks for your review @dsolanki-initos 
What sequence of commits? You mean the order of `pre-commit` and `migration`?



> Functional Test:
> For the products without tracking it is working fine.
> For the products with lot "Auto Fill" functionality seems not working.
> 
> Please have a look at the video link I have prepared
> https://www.awesomescreenshot.com/video/3187440?key=c42c6ab8b14d50af55ab99ffae5f49c8

This is just a configuration in the stock.picking.type.
You need to disable this setting, that's enabled by default:

image

**Comment by dsolanki-initos** [comment]
> What sequence of commits? You mean the order of `pre-commit` and `migration`?

Yes the sequence of Pre-commit and migration.

**Review by dsolanki-initos — APPROVED** [review]

**Comment by dsolanki-initos** [comment]
Hi @ivantodorovich  you can also ask someone to add review to get the PR merged.

**Comment by dsolanki-initos** [comment]
@all Can somone add reviews here keeping the goal to merge the PR.

**Comment by ivantodorovich** [comment]
ping @OCA/logistics-maintainers

**Comment by dsolanki-initos** [comment]
@OCA/logistics-maintainers can somone add reviews here and merge it ?

File: stock_move_line_auto_fill/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2017 ACSONE SA/NV
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Move Line Auto Fill",
+    "summary": "Stock Move Line auto fill",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV," "Tecnativa," "Odoo Community Association (OCA)",
```
Comment (ps-tubtim) [inline stock_move_line_auto_fill/__manifest__.py]:
```suggestion
    "author": "ACSONE SA/NV, Tecnativa, Odoo Community Association (OCA)",
```

**Review by ps-tubtim — COMMENTED** [review]
I have a few comments.

File: stock_move_line_auto_fill/__manifest__.py (hunk 1 above)
Comment (ivantodorovich) [reply stock_move_line_auto_fill/__manifest__.py]: Thanks ! :) it's done

**Review by ps-tubtim — APPROVED** [review]

**Pull request merged**